new.go

```diff
@@ -40,10 +40,9 @@ func reserveSELinuxLabels(store storage.Store, id string) error {
 			} else {
 				b, err := OpenBuilder(store, c.ID)
 				if err != nil {
-					if err == storage.ErrContainerUnknown {
-						continue
-					}
-					return err
+					// Ignore errors since containers can be created by other tools
+					// TODO, we need to read other containers json data to reserve their SELinux labels
+					continue
```

Comment:
Were we getting storage.ErrContainerUnknown here?  The logic in https://github.com/projectatomic/buildah/blob/master/cmd/buildah/common.go#L48 is checking for different errors from `OpenBuilder()`.